test/blockstore-test.js

```diff
@@ -281,12 +281,20 @@ describe('BlockStore', function() {
   });
 
   describe('FileBlockStore (Unit)', function() {
-    const location = '/tmp/.bcoin/blocks';
+    const location = () => {
+      switch(process.platform) {
```

Comment:
nit: spacing to `switch (`, you can grep the codebase to find it's common.

Reply:
Fixed.